deploy/addons/heapster-rc.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  labels:
+    kubernetes.io/cluster-service: 'true'
+    k8s-app: heapster
+    name: heapster
+  name: heapster
+  namespace: kube-system
+spec:
+  replicas: 1
+  selector:
+    kubernetes.io/cluster-service: "true"
+    k8s-app: heapster
+  template:
+    metadata:
+      labels:
+        kubernetes.io/cluster-service: 'true'
+        k8s-app: heapster
+    spec:
+      containers:
+      - name: heapster
+        image: kubernetes/heapster:canary
```

Comment:
So it turns out they have been releasing to GCR but pushing latest to dockerhub through their makefiles.  I opened up an issue here https://github.com/kubernetes/heapster/issues/1384

But we should probably pin to their last release
gcr.io/google_containers/heapster:v1.2.0